kola/tests/misc/ntp.go

```diff
@@ -43,19 +49,29 @@ func NTP(c cluster.TestCluster) {
 	}
 	defer m.Destroy()
 
+	c.MustSSH(m, "sudo systemctl restart systemd-timesyncd.service")
```

Comment:
I think it's better if we make sure that timesyncd is enabled instead of starting it here for the test.

Reply:
Removed the `systemctl restart` command, and changed the list of systemd services so it keeps systemd-timesyncd.